toonz/sources/toonz/viewerpane.cpp

```diff
@@ -588,6 +600,12 @@ void SceneViewerPanel::onSceneChanged() {
   TApp *app         = TApp::instance();
   ToonzScene *scene = app->getCurrentScene()->getScene();
   assert(scene);
+  m_flipConsole->setFrameRate(TApp::instance()
+                                  ->getCurrentScene()
+                                  ->getScene()
+                                  ->getProperties()
+                                  ->getOutputProperties()
+                                  ->getFrameRate());
```

Comment:
I see.  I never really used the combo viewer- so I didn't see it fire on every xsheet change.  I agree - we don't want that.